common/src/main/java/org/apache/gravitino/dto/requests/TableCreateRequest.java

```diff
@@ -160,6 +160,8 @@ public void validate() throws IllegalArgumentException {
           .forEach(
               index -> {
                 Preconditions.checkArgument(index.type() != null, "Index type cannot be null");
+                Preconditions.checkArgument(
```

Comment:
Please change the message in L166 accordingly; it should be `The length of fieldNames should be greater than 0` or something similar.